ios-base/Extensions/UIViewControllerExtension.swift

```diff
@@ -16,4 +16,25 @@ extension UIViewController {
     alert.addAction(UIAlertAction(title: "Ok", style: UIAlertAction.Style.default, handler: handler))
     present(alert, animated: true, completion: nil)
   }
+  
+  func goToScreen(withIdentifier identifier: String,
+                  storyboardId: String? = nil,
+                  modally: Bool = false) {
+    var storyboard = self.storyboard
+    
+    if let storyboardId = storyboardId {
+      storyboard = UIStoryboard(name: storyboardId, bundle: nil)
+    }
+    
+    guard let viewController =
+      storyboard?.instantiateViewController(withIdentifier: identifier) else {
```

Comment:
Maybe you can try to rewrite this method to use the `func instantiateViewController <T: UIViewController>(_ type: T.Type) -> T?` already created in the `UIStoryboardExtension` so we can use the Type instead of the identifier added to the storyboard.

Reply:
I personally prefer to use `instantiateViewController(withIdentifier:` but I cannot see a reason why you shouldn't extend this function to give the choice of using type instead.